db/schema.sql

```diff
@@ -12,6 +12,7 @@ SET client_encoding = 'UTF8';
 SET standard_conforming_strings = on;
 SELECT pg_catalog.set_config('search_path', '', false);
 SET check_function_bodies = false;
+SET xmloption = content;
```

Comment:
What does this do exactly?

Reply:
I think this is just confirming a default for how strings are cast to/from xml: https://www.postgresql.org/docs/current/datatype-xml.html

I don't think this is relevant to our schema, since I can't think of a place where we're casting xml. But it's auto-generated in the schema I get running Postgres 11